experiments/checkpoint_experiment_data.py

```diff
@@ -97,9 +112,38 @@ def _print_map(self):
                 tput = 0
                 lat = 0
                 if measurement:
+                    row_has_data = True
                     tput = measurement.get_non_gpu_metric_value(
                         'perf_throughput')
                     lat = measurement.get_non_gpu_metric_value(
                         'perf_latency_p99')
                 row_str += f"\t{tput:4.1f}:{lat:4.1f}"
-            print(row_str)
+
+            # Print at least the first 7 rows, and then stop when we hit an empty row
+            # (Some databases don't have data for the first 6 rows)
+            if row_has_data or i < 7:
+                print(row_str)
+            else:
+                break
+
+    def _has_exponential_inst_count(self) -> bool:
```

Comment:
Why 3? Just because it's the first non-exponent? If this isn't present does this guarantee all the remaining instances will be exponents?

Reply:
Yeah, in the databases, it is either 1,2,3,4,5,... or 1,2,4,8,16,...
I picked the first one to make sure early exit didn't impact